micro-rdk-nmea/src/messages/pgns.rs

```diff
@@ -164,14 +164,14 @@ macro_rules! define_pgns {
 pub const MESSAGE_DATA_OFFSET: usize = 32;
 
 define_pgns!(
-    (WaterDepth, 128267),
-    (TemperatureExtendedRange, 130316),
-    (GnssSatsInView, 129540)
+    WaterDepth,
+    TemperatureExtendedRange,
+    GnssSatsInView
 );
 
 pub struct NmeaMessage {
-    metadata: NmeaMessageMetadata,
-    data: NmeaMessageBody,
+    pub metadata: NmeaMessageMetadata,
```

Comment:
Needed for testing, let me know if you would prefer accessors, but I find them less interesting in a language where immutability is the default.

Reply:
if it's just for testing, would `pub(crate)` work?

Reply:
Ah, yes, I think it would. I always forget that that is an option. Too accustomed to C++ having only public/private. If it works I'll make that change before push.